addons/im_livechat/static/tests/mock_server/livechat_mock_server.js

```diff
@@ -75,7 +75,6 @@ async function get_session(request) {
                 makeKwArgs({ fields: ["avatar_128", "user_livechat_username"] })
             ),
             name: channelVals["name"],
-            open_chat_window: true,
```

Comment:
I'm confused, this is removed in mock server but not in python?

Reply:
mock server was wrong (python doesn't send it)